lib/matplotlib/axes/_axes.py

```diff
@@ -1408,59 +1408,66 @@ def plot(self, *args, **kwargs):
     def plot_date(self, x, y, fmt='o', tz=None, xdate=True, ydate=False,
                   **kwargs):
         """
-        Plot with data with dates.
+	A plot with data that contains dates.
 
-        Call signature::
-
-           plot_date(x, y, fmt='bo', tz=None, xdate=True,
-                     ydate=False, **kwargs)
-
-        Similar to the :func:`~matplotlib.pyplot.plot` command, except
+	Similar to the func : matplotlib.pyplot.plot command, except
         the *x* or *y* (or both) data is considered to be dates, and the
         axis is labeled accordingly.
 
         *x* and/or *y* can be a sequence of dates represented as float
         days since 0001-01-01 UTC.
 
-        Keyword arguments:
+        Note if you are using custom date tickers and formatters, it
+        may be necessary to set the formatters/locators after the call
+        to meth : plot_date since meth : plot_date will set the
+        default tick locator to
+        class : matplotlib.dates.AutoDateLocator (if the tick
+        locator is not already set to a
+        class : matplotlib.dates.DateLocator instance) and the
+        default tick formatter to
+        class : matplotlib.dates.AutoDateFormatter (if the tick
+        formatter is not already set to a
+        class : matplotlib.dates.DateFormatter instance).
+
 
-          *fmt*: string
-            The plot format string.
+	Parameters
+    	---------- 
+	fmt : string
+             The plot format string.
 
-          *tz*: [ *None* | timezone string | :class:`tzinfo` instance]
+        tz : [ *None* | timezone string | :class:`tzinfo` instance]
             The time zone to use in labeling dates. If *None*, defaults to rc
             value.
 
-          *xdate*: [ *True* | *False* ]
+        xdate : boolean
             If *True*, the *x*-axis will be labeled with dates.
 
-          *ydate*: [ *False* | *True* ]
+        ydate : boolean
             If *True*, the *y*-axis will be labeled with dates.
 
-        Note if you are using custom date tickers and formatters, it
-        may be necessary to set the formatters/locators after the call
-        to :meth:`plot_date` since :meth:`plot_date` will set the
-        default tick locator to
-        :class:`matplotlib.dates.AutoDateLocator` (if the tick
-        locator is not already set to a
-        :class:`matplotlib.dates.DateLocator` instance) and the
-        default tick formatter to
-        :class:`matplotlib.dates.AutoDateFormatter` (if the tick
-        formatter is not already set to a
-        :class:`matplotlib.dates.DateFormatter` instance).
 
-        Valid kwargs are :class:`~matplotlib.lines.Line2D` properties:
+    	Returns
+    	-------
+    	lines
 
-        %(Line2D)s
 
-        .. seealso::
+	See Also
+    	--------
+    	mod : matplotlib.dates for helper functions
+        func : matplotlib.dates.date2num
```

Comment:
There is a strange indentation here.
In addition, we need to revert back to the old syntax: :func:`~matplotlib.dates.date2num` for proper formatting.